modules/dnn/test/test_ie_models.cpp

```diff
@@ -259,5 +259,46 @@ INSTANTIATE_TEST_CASE_P(/**/,
     Combine(testing::ValuesIn(getAvailableTargets(DNN_BACKEND_INFERENCE_ENGINE)), intelModels())
 );
 
+typedef TestWithParam< tuple< Size, Target > > IEBackend_Test;
+TEST_P(IEBackend_Test, Frame_Width_Height_Test)
```

Comment:
I think that it'd be better to replace this test to [test_backends.cpp](https://github.com/opencv/opencv/blob/master/modules/dnn/test/test_backends.cpp).